Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml

```diff
@@ -0,0 +1,67 @@
+- type: entity
+  name: fitz special
+  parent: [BaseWeaponRevolver, BaseC1Contraband]
+  id: WeaponRevolverFitz
+  description: A compact and concealable self defence snub revolver. Uses .38 special ammo.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Objects/Weapons/Guns/Revolvers/fitz.rsi
+  - type: Clothing
+    sprite: DeltaV/Objects/Weapons/Guns/Revolvers/fitz.rsi
+  - type: Gun
+    selectedMode: SemiAuto
+    fireRate: 1.75
+    availableModes:
+    - SemiAuto
+  - type: RevolverAmmoProvider
+    proto: CartridgeMagnum
+    capacity: 6
+    chambers: [ True, True, True, True, True, True ]
+    ammoSlots: [ null, null, null, null, null, null ]
+  - type: StaticPrice
+    price: 340
+
+- type: entity
+  name: faith
+  parent: [BaseWeaponRevolver, BaseC1Contraband]
+  id: WeaponRevolverFaith
+  description: Delivers blessings in bullet form. Uses .38 special ammo.
```

Comment:
Misleading description: It states the gun uses .38 special, which is not present in the Frontier codebase. In reality, it uses .45 magnum.